cluster/gce/util.sh

```diff
@@ -1367,6 +1367,9 @@ KUBE_PKI_READERS_GROUP: 2005
 ETCD_RUNASUSER: 2006
 ETCD_RUNASGROUP: 2006
 KUBE_POD_LOG_READERS_GROUP: 2007
+KONNECTIVITY_SERVER_RUNASUSER: 2008
+KONNECTIVITY_SERVER_RUNASGROUP: 2008
+KONNECTIVITY_SERVER_SOCKET_WRITER_GROUP: 2008
```

Comment:
Do we want to overload 2008? I realize it should just be the KAS and Proxy-Server which have this group but this means we are giving the KAS access to everything secured by the Konnectivity Server group.

Reply:
Wouldn't konnectivity-server need to support taking a flag to specify which group should have ownership to the socket?
If konnectivity-server runs as lets say user 100 and group 100 and creates the socket if it does not exist then, the permissions of  the socket would be 770 100(user) 100(group). Now konnectivity-server would need to chgrp or similar to change the ownership to another group. I don't think that konnectivity-server supports that today.

Reply:
Fair.